Langs/0415.lang

```diff
@@ -64,3 +65,15 @@ Invalid Release Tags Received!=Otrzymano błądne tagi wydań!
 Invalid Release Types Received!=Otrzymano błądne typy wydań!
 Update Available=Aktualizacja dostępna
 An Update is Available, would you like to download it?=Aktualizacja dostępna, chcesz ją pobrać?
+[Descriptions]
+Architecture=Liczba danych jakich twój procesor i system operacyjny może przetworzyć na raz. Systemy 32-bitowe wymagają wyczyszczenia dysku i nowej instalacji Windows 11. 32-bitowe procesory muszą być zastąpione.
+Boot=Metoda jakiej twoja płyta główna używa aby załadować Windows. Wynik legacy może zostać naprawiony na nowszych płytach w ustawianych BIOS/UEFI. Sprawdź instrukcję obsługi twojej płyty głównej.
+CPU Name=Procesor jaki masz zamontowany w twoim komputerze. Kompatybilność może ulec zmianie. Wymaga fizycznej wymiany na komputerach stacjonarnych; nie wymienialne w laptopach.
+CPU Cores=Liczba zadań jakich twój procesor może przetworzyć na raz. Wymaga fizycznej wymiany na komputerach stacjonarnych; nie wymienialne w laptopach.
```

Comment:
Tu to w ogole nie rozumiem oryginalnego tekstu i mi sie nie podoba.
co do tlumaczenia: " zadan ktore" zamiast "jakich"
w zamiast na
niewymienialne

Reply:
Poprawie obydwa bo "które" mi jakoś bardziej zrozumiale brzmi